bld/CLMBuildNamelist.pm

```diff
@@ -3742,8 +3742,8 @@ sub setup_logic_fates {
               $var = "fates_inventory_ctrl_filename";
               if ( ! defined($nl->get_value($var))  ) {
                  $log->fatal_error("$var is required when use_fates_inventory_init is set" );
-              } elsif ( ! -f "$nl->get_value($var)" ) {
-                 $log->fatal_error("$var does NOT point to a valid filename" );
+              # } elsif ( ! -f "$nl->get_value($var)" ) {
+              #    $log->fatal_error("$var does NOT point to a valid filename" );
```

Comment:
My bet is that extra quotes in the string are making the check for file existence to fail. So I think I can fix this so it will work.